iac-extensions/terraform/src/main/java/org/sonar/iac/terraform/checks/gcp/PublicAccessCheck.java

```diff
@@ -71,8 +80,25 @@ protected void registerResourceConsumer() {
       resource -> resource.attribute("visibility")
         .reportIf(equalTo("public"), MESSAGE)
         .reportIfAbsent(OMITTING_DNS));
-  }
 
+    register("google_container_cluster",
+      resource -> {
+        BlockSymbol config = resource.block("private_cluster_config");
+        config.reportIfAbsent(OMITTING_KUBERNETES);
```

Comment:
Missing `String.format(OMITTING_KUBERNETES, "private_cluster_config")` here?

And besides, when it is *Absent* you have to break here, and not go on any further, as doing otherwise will raise quite some noise?